auth-microservice/app/tests/test_users.py

```diff
@@ -4,6 +4,7 @@
 from typing import Dict
 
 from app.tests.factories import UserFactory
+from app.core.config import settings
 
 @pytest.mark.asyncio()
 async def test_get_users_200(client: AsyncSession, session: AsyncConnection, user_token_headers: Dict[str, str]):
```

Comment:
Same as above, you can subscript native `dict`, use that instead
```suggestion
from app.tests.factories import UserFactory
from app.core.config import settings

@pytest.mark.asyncio()
async def test_get_users_200(client: AsyncSession, session: AsyncConnection, user_token_headers: dict[str, str]):
```